chain/chain/src/store.rs

```diff
@@ -546,25 +476,22 @@ impl<'a, T: ChainStoreAccess> ChainStoreUpdate<'a, T> {
         &mut self,
         shard_id: ShardId,
         mut block_hash: CryptoHash,
-        last_chunk_header: &ShardChunkHeader,
-    ) -> Result<Vec<ReceiptResponse>, Error> {
+        last_chunk_height_included: BlockIndex,
+    ) -> Result<Vec<ReceiptProofResponse>, Error> {
         let mut ret = vec![];
 
-        if last_chunk_header.inner.prev_block_hash == CryptoHash::default() {
-            return Ok(ret);
-        }
-
         loop {
             let header = self.get_block_header(&block_hash)?;
 
-            if header.inner.height == last_chunk_header.height_included {
+            // TODO >= <= ?
```

Comment:
?

Reply:
the logic is changed, and now it's not safe to just check equality
I'll discuss it with @SkidanovAlex